src/writer/logs.py

```
@@ -1,104 +1,244 @@
+            data.update(record.args)
+
+        if isinstance(record.args, tuple):
+            if len(record.args) > 0 and isinstance(record.args[0], dict):
+                data.update(record.args[0])
+
+        return json.dumps(data)
+
+
+class LocalFormatter(logging.Formatter):
+    def format(self, record: logging.LogRecord) -> str:
+        super(LocalFormatter, self).format(record)
```

Pylint (3.3.7)

[refactor] Consider using Python 3 style super() without arguments

(R1725)